daos/mainnet/detroit-arcology.json

```diff
@@ -0,0 +1,9 @@
+{
+        "name": "DetroitDAO",
+        "Avatar": "0xf931FF012CBff471CccB90A784E3E88B512c890A",
+        "DAOToken": "0xe6448C6502F41ee2949e0d7Fa7eb3c8846f909F9",
+        "Reputation": "0x68cCB9a589012dB8C697c3e9e1908C2aDEEC4BF4",
+        "Controller": "0x9868f32D9A2C3533b56D718deec68cDDc2E55A51",
```

Comment:
looks like this is a uController (not Controller) .
https://etherscan.io/address/0x9868f32D9A2C3533b56D718deec68cDDc2E55A51#code
right ?
in that case you do not need to mention that on the Dao.json.